lib/auth/refreshclient.js

```diff
@@ -49,8 +49,8 @@ function callback(c, err, res) {
  * @private
  */
 UserRefreshClient.prototype.refreshToken_ = function(ignored_, opt_callback) {
-  UserRefreshClient.super_.prototype.refereshToken_.call(this._refreshToken,
-                                                     opt_callback);
+  UserRefreshClient.super_.prototype.refreshToken_.call(
+      this, this._refreshToken, opt_callback);
```

Comment:
This method was being called in the context of the refresh token. Just needed an extra `this` - let me know if I should format this change differently.